roles/installer/tasks/main.yml

```diff
@@ -4,6 +4,22 @@
     that:
       - foreman_installer_scenario is defined
 
+- when:
+    - ansible_distribution_major_version == '8'
+    - ansible_os_family == 'RedHat'
+  block:
+    - name: Get foreman-installer version
+      package:
+        list: foreman-installer
```

Comment:
`list` is not documented for `package`, just for `yum` and `dnf`, and while this works, I'd prefer to stick to `dnf` here.